.travis.yml

```diff
@@ -127,6 +129,14 @@ script:
     - if [ $ZFS_BUILD_TAGS = 1 ]; then
         ./build_image.sh || travis_terminate 1;
       fi
+    # If this build is running due to travis release tag, and
+    # this job indicates to push the release downstream, then
+    # go ahead and tag the dependent repo.
+    # Note that ZFS_BUILD_TAGs can be enabled for arm and amd and we are setting to downstream
+    # after amd images have been pushed.
```

Comment:
nit: comments. "we are setting to downstream after amd images have been pushed."